usr/share/rear/verify/CDM/default/410_use_replica_cdm_cluster_cert.sh

```diff
@@ -0,0 +1,43 @@
+# 410_use_replica_cdm_cluster_cert.sh
+# If restoring from a replica Rubrik (CDM) cluster use it's cert for RBS.
+
+CDM_RBA_DIR=/etc/rubrik
+CDM_KEYS_DIR=${CDM_RBA_DIR}/keys
+
+# When USER_INPUT_CDM_REPLICA_CLUSTER has any 'true' value be liberal in what you accept and assume exactly 'y' was actually meant:
+LogPrint ""
+is_true "$USER_INPUT_CDM_REPLICA_CLUSTER" && USER_INPUT_CDM_REPLICA_CLUSTER="y"
+while true ; do
+    # Find out if the restore is being done from the original CDM cluster or a Replica
+    # the default (i.e. the automated response after the timeout) should be 'n':
+    answer="$( UserInput -I CDM_REPLICA_CLUSTER -p "Is the data being restored from the original CDM Cluster? (y/n)" -D 'y' -t 300 )"
+    is_true "$answer" && return 0
+    if is_false "$answer" ; then
+        break
+    fi
+    UserOutput "Please answer 'y' or 'n'"
+done
+
+while true; do
+    CDM_CLUSTER_IP="$(UserInput -I USER_INPUT_CDM_CLUSTER_IP -r -t 0 -p "Enter one of the IP addresses for the replica CDM cluster: ")"
+    [[ -n "$CDM_CLUSTER_IP" ]] && break
+    PrintError "Please enter a non-empty CDM cluster IP."
+done
+
+CDM_SUNOS_TAR=rubrik-agent-sunos5.10.sparc.tar.gz
+cd /tmp
+/usr/bin/curl -kLOJ https://${CDM_CLUSTER_IP}/connector/${CDM_SUNOS_TAR} >&2
```

Comment:
@DamaniN 
usually things like `1>&2` are not needed because both stdout and stderr are redirected
into ReaR's log file, cf. "What to do with stdin, stdout, and stderr" in
https://github.com/rear/rear/wiki/Coding-Style

Reply:
Removed the redirects.